Zend/zend_language_parser.y

```diff
@@ -227,16 +227,16 @@ static YYSIZE_T zend_yytnamerr(char*, const char*);
 /* Token used to force a parse error from the lexer */
 %token T_ERROR
 
-%type <ast> top_statement namespace_name name statement function_declaration_statement
+%type <ast> top_statement namespace_name name statement annotated_statement function_declaration_statement
```

Comment:
A bit of terminology mix here with annotations vs attributes.

Reply:
Naming things is hard... would `attributed_statement` be a better fit? In addition there are `annotated_class_statement` and `annotated_parameter` that should also be renamed. Any suggestions as far as new names are concerned?